nixos/modules/programs/iay.nix

```diff
@@ -0,0 +1,37 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.iay;
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption optionalString types;
+in {
+  options.programs.iay = {
+    enable = mkEnableOption (lib.mdDoc "iay");
+    package = mkPackageOption pkgs "iay" {};
+
+    minimalPrompt = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc "Whether to use minimal one-liner prompt.";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    programs.bash.promptInit = ''
+      if [[ $TERM != "dumb" && (-z $INSIDE_EMACS || $INSIDE_EMACS == "vterm") ]]; then
```

Comment:
```suggestion
      if [[ "${TERM}" != 'dumb' && (-z "${INSIDE_EMACS}" || "${INSIDE_EMACS}" == 'vterm') ]]; then
```

Reply:
Will `${}` not be interpreted in Nix in such case?

Reply:
Yes, Nix interprets the strings first.